tests/compliance_operator_v2_test.go

```diff
@@ -190,3 +254,50 @@ func getscanConfigID(configName string, scanConfigs []*v2.ComplianceScanConfigur
 	}
 	return configID
 }
+
+func TestComplianceV2ScheduleRescan(t *testing.T) {
+	conn := centralgrpc.GRPCConnectionToCentral(t)
+	client := v2.NewComplianceScanConfigurationServiceClient(conn)
+	integrationClient := v2.NewComplianceIntegrationServiceClient(conn)
+	resp, err := integrationClient.ListComplianceIntegrations(context.TODO(), &v2.RawQuery{Query: ""})
+	if err != nil {
+		t.Fatal(err)
+	}
+	clusterId := resp.Integrations[0].ClusterId
+
+	scanConfigName := "cis-scan-schedule"
+	sc := v2.ComplianceScanConfiguration{
+		ScanName: scanConfigName,
+		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
+			OneTimeScan: false,
+			Profiles:    []string{"ocp4-cis", "ocp4-cis-node"},
+			ScanSchedule: &v2.Schedule{
+				IntervalType: 3,
+				Hour:         0,
+				Minute:       0,
+			},
+			Description: "Scan schedule for CIS profiles to run daily.",
+		},
+		Clusters: []string{clusterId},
+	}
+	_, err = client.CreateComplianceScanConfiguration(context.TODO(), &sc)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// This is a workaround to get the ID
```

Comment:
Write a ticket to add an endpoint to get by name and/or write one to update the create to return the ID.  Neither are terribly urgent, but we should put them on the list.